templates/erp/includes/accessibilite.html

```diff
@@ -11,7 +11,7 @@
   {% if info.fields %}
     <h2 class="h5 mb-0 pb-1 border-bottom">
       <i aria-hidden="true" class="icon icon-{{ info.icon }} mr-1"></i>
-      <b>{{ section }}</b>
+      <span class="font-weight-bold">{{ section }}</span>
```

Comment:
You're gonna hate me but the class should rather be added to the parent `h3` tag.

Reply:
No problems :) the task wasn't described formally anyways, so I just emphasized the already visible part. Tell me if this if better